docs/source/install.rst

```diff
@@ -53,9 +53,9 @@ Optional Libraries
 Some features in CuPy will only be enabled if the corresponding libraries are installed.
 
 * `cuDNN <https://developer.nvidia.com/cudnn>`_ (library to accelerate deep neural network computations)
-    * Supported Versions: v5, v5.1, v6, v7, v7.1, v7.2, v7.3, v7.4 and v7.5.
+    * Supported Versions: v7, v7.1, v7.2, v7.3, v7.4 and v7.5.
 * `NCCL <https://developer.nvidia.com/nccl>`_  (library to perform collective multi-GPU / multi-node computations)
-    * Supported Versions: v1.3.4, v2, v2.1, v2.2, v2.3 and v2.4.
+    * Supported Versions: v2, v2.1, v2.2, v2.3 and v2.4.
```

Comment:
The minimum version of NCCL that run on CUDA 9.0 is v2.

Reply:
Great! We can remove a lot of compatibility code for NCCL v1.x later!